sormas-ui/src/main/java/de/symeda/sormas/ui/SormasErrorHandler.java

```diff
@@ -83,6 +85,8 @@ public static void handleError(ErrorEvent event) {
 			} else {
 				logger.warn(t.getMessage(), t);
 			}
+
+			t.printStackTrace();
```

Comment:
Why is this needed? As far as I can tell it leads to the error message being output twice.

Reply:
For me it doesn't log the exception, that's why I always add printStackTrace and forget to remove before commit :)

I will remove it